package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@umbraco/playwright-testhelpers",
-  "version": "2.0.0-beta.12",
+  "version": "2.0.0-beta.13",
```

```suggestion
  "version": "2.0.0-beta.14",
```

There is already a version 2.0.0-beta.13 released